x-pack/solutions/observability/plugins/infra/test/scout/ui/parallel_tests/inventory/inventory.spec.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from '@kbn/scout-oblt';
+import { test } from '../../fixtures';
+
+test.describe('Infrastructure Inventory', { tag: ['@ess', '@svlOblt'] }, () => {
+  test('Page should load', async ({ pageObjects: { inventoryPage }, browserAuth, page }) => {
+    await browserAuth.loginAsViewer();
+    await inventoryPage.goto();
+
+    await expect(page.getByTestId('breadcrumb last')).toHaveText('Infrastructure inventory');
```

Comment:
I would check the title instead, but I know that it's just a boilerplate, and we will update that once we have the actual test/data